lib/doorkeeper/config.rb

```diff
@@ -115,9 +115,13 @@ def enable_pkce_without_secret
         @config.instance_variable_set(:@pkce_without_secret_enabled, true)
       end
 
-      # Issue access tokens with refresh token (disabled by default)
-      def use_refresh_token
-        @config.instance_variable_set(:@refresh_token_enabled, true)
+      # Issue access tokens with refresh token (disabled if not set)
+      def use_refresh_token(enabled = true, &custom)
```

Comment:
I added the `enabled = true` parameter here in case people do `use_refresh_token false`. Since this takes a block parameter, I thought to make it a bit consistent.

Reply:
I prefer `&block` instead of `&custom` as a common practice, what do you think?

Also check out `force_ssl_in_redirect_uri` option and [how it works with block or boolean variables](https://github.com/doorkeeper-gem/doorkeeper/blob/master/app/validators/redirect_uri_validator.rb#L36-L43).